tests/Unit/Core/Domain/Order/QueryResult/OrderInvoiceAddressForViewingTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Tests\Unit\Core\Domain\Order\QueryResult;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderInvoiceAddressForViewing;
+
+class OrderInvoiceAddressForViewingTest extends TestCase
+{
+    public function testConstruct(): void
+    {
+        $instance = new OrderInvoiceAddressForViewing(1, 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k');
+        $this->assertEquals(1, $instance->getAddressId());
+        $this->assertEquals('a b', $instance->getFullName());
+        $this->assertEquals('c', $instance->getCompanyName());
```

Comment:
imho the only method worth checking was `getFullName` :D  Overall, since the class has no error checking or value checking there's not much logic to test...  (same for `tests/Unit/Core/Domain/Order/QueryResult/OrderShippingAddressForViewingTest.php` )